.eslintrc.cjs

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2024: true,
```

Comment:
The `env` property specifies `es2024`, but `ecmaVersion` in `parserOptions` is set to 12, which corresponds to ES2021. Ensure these settings are consistent.